plugins/ptp_operator/metrics/metrics.go

```diff
@@ -308,11 +308,16 @@ func (p *PTPEventManager) ExtractMetrics(msg string) {
 		case ClockRealTime:
 			UpdatePTPMetrics(phc, processName, interfaceName, ptpOffset, maxPtpOffset, frequencyAdjustment, delay)
 		case MasterClockType:
-			UpdatePTPMetrics(master, processName, interfaceName, ptpOffset, maxPtpOffset, frequencyAdjustment, delay)
-		case "":
-			//do nothing
-		default:
-			UpdatePTPMetrics(phc, processName, interfaceName, ptpOffset, maxPtpOffset, frequencyAdjustment, delay)
+			ptpInterface, _ = ptp4lCfg.ByRole(types.SLAVE)
+			var alias string
+			if ptpInterface.Name != "" {
+				r := []rune(ptpInterface.Name)
+				alias = string(r[:len(r)-1]) + "x"
```

Comment:
is it possible to have a double digits index number a the end?

Reply:
This is for network card index so one card won't have 10 ports.